samples/subscriptions.js

```diff
@@ -468,49 +403,48 @@ function listenForOrderedMessages(subscriptionName, timeout) {
   };
 
   // Listen for new messages until timeout is hit
-  return new Promise(resolve => {
+  await new Promise(resolve => {
```

Comment:
We should switch this over to a full async function, and use the `await new Promise(r => setTimeout(r, xxx))` pattern